FHEM/00_SIGNALduino.pm

```diff
@@ -4847,75 +4847,93 @@ With a # at the beginnging whitelistIDs can be deactivated.
 	<a name="SIGNALduinoget"></a>
 	<b>Get</b>
 	<ul>
-    <a name="availableFirmware"></a>
-    <li>availableFirmware<br>
-	Ruft die verf&uuml;gbaren Firmware-Version von github ab und macht diese im set flash Befehl ausw&auml;hlbar.</li><br><br>
+    	<a name="availableFirmware"></a>
+    	<li>availableFirmware<br>
+	Ruft die verf&uuml;gbaren Firmware-Version von github ab und macht diese im set flash Befehl ausw&auml;hlbar.
+	</li><br>
 	<a name="ccconf"></a>
-	<li>ccconf<br></li>
+	<li>ccconf<br>
    Liest s&auml;mtliche radio-chip (cc1101) Register (Frequenz, Bandbreite, etc.) aus und zeigt die aktuelle Konfiguration an.<br>
-   (NUR bei Verwendung eines cc1101 Empf&auml;nger)<br><br>
+   (NUR bei Verwendung eines cc1101 Empf&auml;nger)
+   	</li><br>
```

Comment:
Hintergrundinfo zur evtl ergänzung:
ccconf, ccreg und ccpatable kann nur ausgelesen werden, wenn ein cc1101 erkannt wurde.
Ein cc1101 wurde erkannt, wenn in version "cc1101" enthalten ist.